stdlib/Distributed/src/managers.jl

```diff
@@ -462,10 +462,29 @@ function launch(manager::LocalManager, params::Dict, launched::Array, c::Conditi
     exename = params[:exename]
     exeflags = params[:exeflags]
     bind_to = manager.restrict ? `127.0.0.1` : `$(LPROC.bind_addr)`
+    env = Dict{String,String}(params[:env])
+
+    # JULIA_(LOAD|DEPOT)_PATH are used to populate (LOAD|DEPOT)_PATH on startup,
```

Comment:
Could we factor this out? We will need to add this to a whole host of other ClusterManager.

My other worry is that this is now inconsistent. Right now the situation is consistently bad, after this code that works well locally might no longer use once you start adding using the SSHManager. Could you add a documentation  section for this behaviour and the SSHManager behaviour?

Reply:
> Could we factor this out? We will need to add this to a whole host of other ClusterManager.

Sure, a function that returs the necessary env dict to replicate the package env?

> My other worry is that this is now inconsistent. [...]

Yea I guess. In my opinion this could be updated for `SSHManager` too. The clusters I have experience with share filesystem anyway. `SSHManager` already assumes that the path to the `julia` binary is the same on the remote, and that it is possible to `cd` to the same directory for example, so for remotes that don't share a fs you already now need to do some extra configuration.

Reply:
I added docs, and will try something out for SSHManager in a separate PR.